.evergreen/config/functions.yml

```diff
@@ -209,6 +218,33 @@ functions:
           TESTS=${TESTS} \
           sh ${PROJECT_DIRECTORY}/.evergreen/run-tests.sh
 
+  "get aws auth secrets":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${aws_test_secrets_role}
```

Comment:
Note: I defined this in the [mongo-php-driver repo vars](https://spruce.mongodb.com/project/63c7aa4432f4171734d487e7/settings/variables) in Evergreen. I _think_ the remaining `iam_auth_` vars are no longer relevant, but that should be confirmed. I was previously using them in an earlier iteration of this PR _before_ switching to use the [AWS Secrets Manager](https://wiki.corp.mongodb.com/display/DRIVERS/Using+AWS+Secrets+Manager+to+Store+Testing+Secrets) (link points to PyMongo wiki).